python/lsst/ip/isr/isrTask.py

```diff
@@ -426,6 +431,8 @@ def run(self, ccdExposure, bias=None, linearizer=None, dark=None, flat=None, def
 
         if self.config.doAssembleCcd:
             ccdExposure = self.assembleCcd.assembleCcd(ccdExposure)
+            if not ccdExposure.getWcs() and self.config.expectWcs:
```

Comment:
It doesn't really matter, but I'd reverse the order here to short-circuit on getting the WCS if it's not necessary.